### DISCUSSION TITLE
ios silent notification permission

### DISCUSSION THREAD

**Discussion by zverexe** [invertase/react-native-firebase#5682] [General]
Hi, I went through the documentation but I am still curious about info re permissions on ios when I receiving silent(data only) notification. I cannot understand if, I can receive a silent notification without permissions on ios? When I received such notification will ios asks me to give permission to that app or it will go without permission request? For example I need to run some request based on such notification

**Comment by mikehardy** [comment]
Hi there! 

https://stackoverflow.com/questions/46727768/do-silent-notifications-require-asking-the-user-for-notification-permissions#50622867

It appears if your remote notifications (FCM) are data-only and do not generate any user interaction / user visible thing, then you do not need permissions from the user.

**Reply by zverexe** [reply]
@mikehardy thank you very much for a fast reply!
